client/lib/wp/browser.js

```diff
@@ -66,4 +67,4 @@ export default wpcom;
 /**
  * Expose `wpcomJetpackLicensing` which uses a different auth token than wpcom.
  */
-export const wpcomJetpackLicensing = wpcomUndocumented( wpcomProxyRequest );
+export const wpcomJetpackLicensing = wpcomUndocumented( wpcomXhrRequest );
```

Comment:
This should use the already imported `wpcomXhrWrapper` instead. The wrapper loads the actual `wpcom-xhr-request` library asynchronously as a separate webpack chunk.

Otherwise, this patch re-introduces the circular dependency issue that @sgomes fixed in #42785.

Reply:
Thanks for the comment, @jsnajdr!

I'll add that circular dependencies tend to be rather insidious, where sometimes everything appears to be working fine, but all of a sudden a seemingly unrelated change modifies the import order and everything ends up breaking. Reusing `wpcomXhrWrapper` should be functionally equivalent with no further changes 👍

Reply:
Does it make sense to change it [here](https://github.com/Automattic/wp-calypso/blob/trunk/client/lib/wp/node.js#L25) and [here](https://github.com/Automattic/wp-calypso/blob/trunk/client/lib/wp/node.js#L11) as well?

Reply:
> Does it make sense to change it here and here as well?

No, that's server side code running in Node.js, in a completely different environment without webpack async loading. Only the `lib/wp/browser.js` module needs to use the async-loading wrapper.

Reply:
I've changed the other one